go.mod

```diff
@@ -1,7 +1,5 @@
 module github.com/luna-duclos/instrumentedsql
 
-go 1.12
```

Comment:
Nice try, but the next time you run the `go` tool, it will re-add the statement. Note that the meaning of this statement isn't "we only support this specific version". It means "we support any version starting at X", and even then it is only advisory. It is not normally checked for (so Go 1.11 could build this module just fine). Only when compilation fails will Go consult this statement, to print a helpful message of this kind: 

```
 go build ./foo
# honnef.co/go/tools/foo
foo/foo.go:4:7: syntax error: unexpected l000 at end of statement
note: module requires Go 1.14
```

Ideally, you'd set this to the lowest supported Go version. Practically, leaving it at 1.12 is just fine, too.